adhocracy4/polls/assets/PollDashboard/EditPollManagement.jsx

```diff
@@ -195,7 +194,7 @@ export const EditPollManagement = (props) => {
       onSubmit={(e) => handleSubmit(e)} onChange={() => removeAlert()}
       className="editpoll__questions"
     >
-      <FlipMove easing="cubic-bezier(0.25, 0.5, 0.75, 1)">
+      <FlipMove easing="cubic-bezier(0.25, 0.5, 0.75, 1)" typeName={null}>
```

Comment:
`typeName={null}` is this really needed?

Reply:
okay, that gets removed in a later commit

Reply:
missed that when cleaned commits but will leave it if cool?